src/modules/dashboard/components/Dialogs/ControlSafeDialog/ControlSafeDialog.tsx

```diff
@@ -35,8 +35,8 @@ export interface SafeTransaction {
   nft: SelectedNFT | null;
   nftData: NFT | null;
   tokenData: SafeBalanceToken | null;
-  amount: number | null;
-  rawAmount: number | null;
+  amount: string | null;
```

Comment:
These are coming direct from the input as strings, which is actually ok because ethers accepts number strings for uint input types. This is convenient in the raw transaction section for example, where the amount of wei entered can exceed the max safe integer in javascript. Passing it through straight as a string fixes overflow errors.